### PR TITLE
SOV-3777: boltz v2 api

### PR DESCRIPTION
**Pull request by creed-victor**
https://sovryn.atlassian.net/browse/SOV-3777

### CODE REVIEW

File: apps/frontend/src/app/3_organisms/FastBtcDialog/utils/boltz/boltz.client.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  SubmarineSwapResponse,
+  SubmarineSwapTransactionResponse,
+  SwapResponse,
+} from './boltz.types';
+
+const baseUrl = isMainnet()
+  ? 'https://api.boltz.exchange/api/v2'
+  : 'https://testnet.boltz.exchange/api/v2';
+
+const client = Axios.create({
```
Comment (rick23p) [inline apps/frontend/src/app/3_organisms/FastBtcDialog/utils/boltz/boltz.client.ts:28]: boltzClient? and doesn't need to be exported

File: apps/frontend/src/app/3_organisms/FastBtcDialog/utils/boltz/boltz.types.ts [hunk 2]
```diff
@@ -0,0 +1,183 @@
+export enum StatusEnum {
```
Comment (rick23p) [inline apps/frontend/src/app/3_organisms/FastBtcDialog/utils/boltz/boltz.types.ts]: I think this can be better named

**Review by rick23p — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/3_organisms/FastBtcDialog/utils/boltz/boltz.client.ts (hunk 1 above)
Comment (creed-victor) [reply apps/frontend/src/app/3_organisms/FastBtcDialog/utils/boltz/boltz.client.ts:28]: It's being wrapped by functions in this file and does not used for outside it, so i don't think we need need it exported

File: apps/frontend/src/app/3_organisms/FastBtcDialog/utils/boltz/boltz.types.ts (hunk 2 above)
Comment (creed-victor) [reply apps/frontend/src/app/3_organisms/FastBtcDialog/utils/boltz/boltz.types.ts]: Open to suggestions, just that StatusType are already used in the fastbtc flows

Comment (rick23p) [reply apps/frontend/src/app/3_organisms/FastBtcDialog/utils/boltz/boltz.types.ts]: Maybe `BoltzTxStatus`

**Review by rick23p — APPROVED** [review]

**Pull request merged**